src/common/Header/Logo/index.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 import { Icon, LogoWrapper } from "./styled";
 import LogoImage from "../../../assets/images/logo.svg";
+import { Link } from "react-router-dom";
+import { toMovies } from "../../../routes";
 
 const Logo = () => (
-    <LogoWrapper>
+    <LogoWrapper as={Link} to={toMovies()}>
```

Comment:
Może lepiej to={"/"}? W naszym przypadku zadziała tak samo bo "/" przekierowuje na movies, ale będzie bardziej porządek że logo przenosi do strony głównej i jeśli kiedykolwiek taka powstanie (chociaż nie zakładam że tak będzie) to dalej będzie działać

Reply:
+1